source/agora/common/Config.d

```diff
@@ -276,6 +276,9 @@ public struct AdminConfig
     /// Is the control API enabled?
     public bool enabled;
 
+    /// Whether to enforce tls is configured for admin
+    public bool enforce_tls = false;
```

Comment:
Well the issue referenced is about allowing the admin interface to be HTTP even if SSL certificates are present, not soft-failing.

Reply:
I should have tested it locally. I have updated to have `tls` as an option on `interfaces` and on admin.